### ISSUE TITLE
"parquet-cat -j" doesn't show all records

### ISSUE THREAD

**Issue by asfimport** [apache/parquet-java#1883]
Inspect the attached Parquet file generated by parquet-proto to reproduce this issue:
```
$ parquet-cat old-repeated-int.parquet
repeatedInt = 1
repeatedInt = 2
repeatedInt = 3

$ parquet-cat -j old-repeated-int.parquet
{"repeatedInt":3}
```
Expected output should be something like
```
{"repeatedInt":[1,2,3]}
```
Schema of the attached testing Parquet file is:
```
message TestProtobuf.RepeatedIntMessage {
  repeated int32 repeatedInt;
}
```

**Reporter**: [Cheng Lian](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=lian+cheng) / @liancheng
**Assignee**: [Reuben Kuhnert](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sircodesalot) / @sircodesalotOfTheRound
#### Related issues:
- [Release Parquet-mr 1.9.0](https://issues.apache.org/jira/browse/PARQUET-392) (blocks)
#### Original Issue Attachments:
- [old-repeated-int.parquet](https://issues.apache.org/jira/secure/attachment/12752749/old-repeated-int.parquet)
#### PRs and other links:
- [PR-281](https://github.com/apache/parquet-mr/pull/281)

<sub>**Note**: *This issue was originally created as [PARQUET-367](https://issues.apache.org/jira/browse/PARQUET-367). Please see the [migration documentation](https://issues.apache.org/jira/browse/PARQUET-2502) for further details.*</sub>

**Comment by asfimport** [comment]
[Reuben Kuhnert](https://issues.apache.org/jira/browse/PARQUET-367?#comment-14960968) / @sircodesalotOfTheRound:
Added pull request.

**Comment by asfimport** [comment]
[Julien Le Dem](https://issues.apache.org/jira/browse/PARQUET-367?#comment-15009311) / @julienledem:
thanks @sircodesalotOfTheRound. Could you update the description to add what's expected?
I assume something like ```
{"repeatedInt":[1,2,3]}
``` ?

**Comment by asfimport** [comment]
[Cheng Lian](https://issues.apache.org/jira/browse/PARQUET-367?#comment-15010806) / @liancheng:
Updated.

**Comment by asfimport** [comment]
[Ryan Blue](https://issues.apache.org/jira/browse/PARQUET-367?#comment-15273078) / @rdblue:
Merged #281. Thanks for fixing this, @sircodesalotOfTheRound!
